src/Neo.VM/Unsafe.cs

```diff
@@ -38,5 +41,16 @@ public static bool NotZero(ReadOnlySpan<byte> x)
             }
             return false;
         }
+
+        /// <summary>
+        /// Get 64-bit hash code for a byte array
+        /// </summary>
+        /// <param name="span">Span</param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong HashBytes(ReadOnlySpan<byte> span)
```

Comment:
It might be better to move this into `Neo.Extension.ByteExtension`.

Reply:
We can do it in the next pr, this is almost approved and tested